airflow/utils/db.py

```diff
@@ -120,6 +120,9 @@ def initdb():
             conn_id='bigquery_default', conn_type='bigquery'))
     merge_conn(
         models.Connection(
+            conn_id='google_cloud_default', conn_type='google_cloud_platform'))
```

Comment:
Yes these are duplicate code when PR2379 was not merged into Airflow. I will get rid of those.
Thank you!